test/storage-luatest/storage_persistent_names_test.lua

```diff
@@ -0,0 +1,118 @@
+local t = require('luatest')
```

Comment:
No need to call the file `storage-luatest/storage_persistent_names_test.lua` - it is in the storage folder anyway. Can be called just `storage-luatest/persistent_names_test.lua`.

Reply:
Done